.promu.yml

```diff
@@ -12,6 +12,7 @@ build:
         -X github.com/prometheus/common/version.Branch={{.Branch}}
         -X github.com/prometheus/common/version.BuildUser={{user}}@{{host}}
         -X github.com/prometheus/common/version.BuildDate={{date "20060102-15:04:05"}}
+        -s -w -extldflags "-static"
```

Comment:
This should probably match the upstream [promu config](https://github.com/prometheus/promu/blob/master/.promu.yml#L4-L6).

Reply:
yep, you were right about the default static. The config now matches the upstream.